pages/agent/v3/gcloud.md.erb

```diff
@@ -177,19 +189,20 @@ spec:
   containers:
   - ...
     volumeMounts:
-    - name: ssh-keys
+    - name: buildkite-agent-ssh
       mountPath: /root/.ssh/id_rsa
       subPath: id_rsa
-    - name: ssh-keys
+    - name: buildkite-agent-ssh
       mountPath: /root/.ssh/id_rsa.pub
       subPath: id_rsa.pub
     ...
   volumes:
-  - name: ssh-keys
+  - name: buildkite-agent-ssh
```

Comment:
the name of these volumes can be anything, so it'd be fine to keep them as `ssh-keys` if you were keen to keep the diff of this PR minimal.

Reply:
Thanks @yob !